pkg/collect/logs.go

```diff
@@ -264,6 +264,18 @@ func setLogLimits(podLogOpts *corev1.PodLogOptions, limits *troubleshootv1beta2.
 	} else {
 		podLogOpts.TailLines = &limits.MaxLines
 	}
+
+	defaultMaxBytes := int64(5000000)
+	if limits.MaxBytes == 0 {
+		podLogOpts.LimitBytes = &defaultMaxBytes
+		return
+	}
+
+	if limits.MaxBytes == 0 {
```

Comment:
What is you have a negative value (`limits.MaxBytes = -200`)

Reply:
This does not affect the working, logs are created regardless of the negative value.